docs/releases/2.0.2.txt

```diff
@@ -14,3 +14,6 @@ Bugfixes
 
 * Fixed incorrect foreign key nullification if a model has two foreign keys to
   the same model and a target model is deleted (:ticket:`29016`).
+
+* Fixed crash in ``QuerySet`` with multiple annotations when using geometry
```

Comment:
Fixed a regression where a queryset that annotates with geometry objects crashes